wire-webapp/content/translation/ar-SA.json

```diff
@@ -681,6 +706,7 @@
   "participantDevicesSelfAllDevices": "أظهر جميع الأجهزة الخاصة بي",
   "participantDevicesSelfFingerprint": "بصمة الجهاز",
   "participantDevicesWhyVerify": "لِمَ أتحقق من مُحادثاتي؟",
+  "prefenrencesAccountResetAppLockPassphrase": "Reset app lock passphrase",
```

Comment:
`prefenrencesAccountResetAppLockPassphrase`

Is the typo in the webapp present, too?